src/main/java/com/streisky/virtualshowcasebackend/controller/product/ProductController.java

```diff
@@ -0,0 +1,42 @@
+package com.streisky.virtualshowcasebackend.controller.product;
+
+import com.streisky.virtualshowcasebackend.dto.product.ProductDTO;
+import com.streisky.virtualshowcasebackend.service.product.ProductService;
+import jakarta.transaction.Transactional;
+import jakarta.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.web.PageableDefault;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("product")
```

Comment:
By the [REST convention](https://blog.dreamfactory.com/best-practices-for-naming-rest-api-endpoints/), endpoint names should be plurals.

Reply:
Updated.